### ISSUE TITLE
Easy installation of an jupyter shortcut

### ISSUE THREAD

**Issue by jlperla** [JuliaLang/IJulia.jl#867]
For those who never use a terminal or a repl, jupyter is an easy way to get started.  However, to actually run jupyter isn't so easy.   @stevengj would you be willing to accept a PR that adds in the ability to generate a jupyter shortcut with IJulia (in one form or another)?  

This could be triggered by a separate function (e.g. `IJulia.createshortcut()` which we could tell users to execute after installation.  Or it could be an argument, e.g. `IJulia.jupyterlab(createshortcut=true)`

What would it do?
On windows, this would run the following powershell script (which works as is, just modify the exelocation):

```powershell
$linklocation= "$HOME/Desktop/jupyter.lnk"
$exelocation = "C:\Users\USERNAME\AppData\Local\Julia-1.2.0\bin\julia.exe"
$WshShell = New-Object -comObject WScript.Shell
$Shortcut = $WshShell.CreateShortcut($linklocation)
$Shortcut.TargetPath = $exelocation
$Shortcut.Arguments = "-e `"using IJulia; jupyterlab()`""
$Shortcut.WorkingDirectory = "$HOME"
$Shortcut.Save()
```
where the linklocation would be an argument, but default to the home desktop, the exelocation could be figured out within julia with `Sys.BINDIR`.

Alternatively, instead of using the `Julia` binary, it could find the `jupyter` binary for the exelocation and adjust the arguments accordingly.

For OSX, it could create a symlink or maybe shell script in the Applications folder?

**Comment by stevengj** [comment]
I see the appeal here.   But wouldn't this shortcut be somewhat problematic if it were clicked multiple times, launching multiple `jupyter` servers simultaneously?

**Comment by stevengj** [comment]
Note that if you want a double-clickable program, that's exactly what nteract is for.

**Comment by jlperla** [comment]
No, I don't want a double-clickable single notebook, but easier launching of the whole jupyterlab environment for those installing from IJulia instead of their own anaconda.

i.e. it basically does `julia -E "using IJulia; jupyterlab()" in a terminal, or just calls the jupyter in conda directly.

Easier discovery and accessibility for matlab-type who only use terminals rarely.  Windows and low-tech OSX users are used to double-clicking on applications.

Jupyterlab remembers the last folder used/etc.  so it is a very good experience.  Multiple notebook servers running is the preference.

**Comment by janxkoci** [comment]
> Is there a way that we can install the Anaconda version? Is this just some conda package?

I think this is the Anaconda Navigator. I don't know what it depends on though, but I have seen people on the internets installing it in miniconda setups, so maybe it's not too heavy..
